app/controllers/hyrax/dashboard/collections_controller.rb

```diff
@@ -120,16 +120,6 @@ def after_destroy_success(change_set)
         end
       end
 
-      def after_destroy_error(id)
```

Comment:
Is this no longer called? If not, how are we notifying the user that something went wrong?

Reply:
I don't believe that there was ever a way to get into this bit of code.  There are no destroy validations on the model. We only had test coverage of this method because we were mocking the response here: https://github.com/samvera/hyrax/blob/master/spec/controllers/hyrax/dashboard/collections_controller_spec.rb#L315